static/app/views/ddm/widget.tsx

```diff
@@ -393,6 +410,25 @@ const MetricWidgetBody = memo(
       [router]
     );
 
+    const handleRowFilter = useCallback(
+      series => {
+        const queryName = getQueryName(series.id);
```

Comment:
imho, we should rather add a `queryName`  property on the series.
relying on the `-` being in the id is rather brittle.

Or even better, the index of the query, so we do not have to search it anymore.

Reply:
make sense, added it in [2b55772](https://github.com/getsentry/sentry/pull/66776/commits/2b55772efe73d81a09525fe43a5bdffbd2c6d8f8)